pkg/render/render_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package render
```

Comment:
Let's use different package for tests to explicitly include stuff we want to test "render_test"

Reply:
I'd love to test it as a proper library. But for e.g. tests under pkg/network/ we cannot really do it, they have private functions like validate and more. Actually, let me change it here and then rework pkg/network to be public and properly organized (and tested) later.

Reply:
Done